cosi/threshold/handler.go

```diff
@@ -1,56 +1,59 @@
+// This file contains the implementation of the RPC handler.
```

Comment:
is this comment accurate? It feels - at the very least - a bit incomplete

Reply:
Hm no, it's really only doing this. It implements the Stream handler for the RPC instantiated by the module.